charter.md

```diff
@@ -53,15 +53,130 @@ The following responsibilities belong to the Steering Committe, but may be deleg
 * Control and delegate access to and establish processes regarding
   project resources/assets<sup>[6](#footnote6)</sup>  
 
-## Quorum
+## Changes
 
-Steering committee members are generally expected to attend every meeting. We
+In instances where a process is not already specified within this document,
+changes to the Steering Committee charter will be considered according to the
+processes set forth in the committee's [operations documentation][changes].
+
+[changes]: /operations/changes.md
+
+## Membership
+
+### Elections
+
+Every year, the Steering Committee holds a general election for open seats with
+[SIG Contributor Experience][sig-contribex] owning the process.
+
+SIG Contributor Experience may delegate this responsibility to a subproject.
+
+Our [election policy document][general-elections] covers the details for how
+this works.
+
+Policy and operations of special elections initiated by the Steering Committee
+for other reasons (as yet to be determined) may be handled by some agreed-upon
+[process][changes] by the committee.
+
+[general-elections]: /elections.md
+[sig-contribex]: https://git.k8s.io/community/sig-contributor-experience
+
+### Vacancies
+
+In the event of a resignation, removal, or other loss of an elected Steering 
+Committee member, the candidate with the next most votes from the previous 
+election will be offered the seat. This process will continue until the seat is 
+filled. 
+
+In case this fails to fill the seat, a special election for that position will 
+be held as soon as possible. [Eligible voters](https://git.k8s.io/steering/elections.md#eligibility-for-voting) from the most recent election 
+will vote in the special election (ie: eligibility will not be redetermined at 
+the time of the special election). A committee member elected in a special 
+election will serve out the remainder of the term for the person they are 
+replacing, regardless of the length of that remainder.
+
+### Resignation
+
+If a committee member chooses not to continue in their role, for whatever
+self-elected reason, they must notify the committee in writing. As a courtesy,
+such notifications should be given at least 30 calendar days in advance of
+their departure.
+
+The committee can choose to accept a member's resignation and carry out
+offboarding procedures at any time following the written notice.
+
+### Removal
+
+#### No confidence
+
+A Steering Committee member may be removed by a two-thirds vote of no
```

Comment:
After thinking about this overnight, I think this should require unanimity (sans target). Since it is overriding the will of the voters. Requiring unanimity will make abuse of this process drastically harder while not making it that much harder to eject someone who definitely deserves it.

Reply:
The other logical choice would be to copy the 3/4ths threshold (presumably rounded up, although not stated) from the [change process](https://github.com/kubernetes/steering/blob/main/operations/changes.md). That works out to the same number for 7 seats but might make more sense for larger numbers of seats.

I do not think it should be easier to eject a board member than it is to change the rules.

Reply:
I think I agree with the 3/4ths suggestion.

While in theory I would prefer unanimity (we strive for all steering decisions to be a consensus whenever possible), I think the logistics make it difficult. Some folks may wish to abstain (such as if the person being removed works for the same employer), and it's just not great practice to give a single person on the committee a veto over any decision (which would be the result of a requirement of unanimity).

Reply:
7 * 3/4 = 5.25, round up is 6. So for this board size it doesn't change the number of votes.

I agree in general, the board should be able to function with a single rogue actor. But for the specific question of ejecting someone, I personally want the rest of the board to be unified about that.

I also think the general threshold should be 50% + 1, not 2/3rds -- then 3/4ths would be a more meaningful step up.

> we strive for all steering decisions to be a consensus whenever possible

It's a good aspiration! But the rules of course need to be written assuming that it's not working and people are going to have to vote on something important to them. If that was an achievable goal, we wouldn't need rules at all. Hence why e.g. Robert's Rules assume fairly antagonistic conditions, I guess.